iXBRLViewerPlugin/viewer/src/js/search.js

```diff
@@ -35,7 +35,14 @@ ReportSearch.prototype.buildSearchIndex = function () {
         doc.startDate = f.periodFrom();
         var dims = f.dimensions();
         for (var d in dims) {
-            l += " " + this._report.getLabel(dims[d],"std");
+            if (this._report.getConcept(d).isTypedDimension) {
+                if (dims[d] !== null) {
+                    l += " " + d;
```

Comment:
Why dimension concept rather than member for typed?  Alternatively, why not dimension concept also for explicit?  If keeping dimension concept, why exclude if nil?

Reply:
This was a typo - it should be the dimension value.